### ISSUE TITLE
PS issue

### ISSUE THREAD

**Issue by Phatdad76** [PrestaShopCorp/ps_accounts#450]
**Issue Summary**  
- A brief description of the issue or request.

**Expected Behavior**  
- Describe what you expected to happen or the outcome you’re hoping for.

**Current Behavior**  
- Outline what actually happens and how it differs from your expectations.

**Steps to Reproduce**  
1. Step-by-step instructions to reproduce the issue.
2. Include any relevant configuration settings or code examples if applicable.

**Environment Information**  
- **PrestaShop Version**: (e.g., 1.7, 8, 9)
- **PHP Version**: (e.g., 7.4, 8.1)
- **Module Version**: (if applicable)

**Screenshots or Logs**  
- Attach any relevant screenshots, error messages, or logs to help diagnose the issue.

**Additional Context**  
- Add any other context about the problem here, including potential solutions or suggestions.

**Comment by Phatdad76** [comment]
Unknown error on line 274 in file /homepages/15/d4298545896/htdocs/clickandbuilds/dronenutter/modules/ps_accounts/vendor/monolog/monolog/src/Monolog/Logger.php
[8192] DateTime::__construct(): Passing null to parameter #1 ($datetime) of type string is deprecated

**Comment by Phatdad76** [comment]
Someone help please, I have this error, when I login to back office.  I can see its a module, but not sure which one.

**Comment by hschoenenberger** [comment]
Hello @Phatdad76 we have a fix for this almost ready, can you tell us if debug mode is activated on your shop and could you add a screenshot ?

**Comment by bluegifts** [comment]
Happened also on ps 8.2 and php 8.1.2
this appears in php error log on server:
[21-Nov-2024 10:16:53 Europe/Bucharest] PHP Deprecated:  DateTime::__construct(): Passing null to parameter #1 ($datetime) of type string is deprecated in /public_html/modules/ps_accounts/vendor/monolog/monolog/src/Monolog/Logger.php on line 274
[21-Nov-2024 10:16:53 Europe/Bucharest] PHP Stack trace:
[21-Nov-2024 10:16:53 Europe/Bucharest] PHP   1. {main}() /public_html/adminxxx/index.php:0
[21-Nov-2024 10:16:53 Europe/Bucharest] PHP   2. Symfony\Component\HttpKernel\Kernel->handle($request = class Symfony\Component\HttpFoundation\Request { public $attributes = class Symfony\Component\HttpFoundation\ParameterBag { protected $parameters = [...] }; public $request = class Symfony\Component\HttpFoundation\ParameterBag { protected $parameters = [...] }; public $query = class Symfony\Component\HttpFoundation\ParameterBag { protected $parameters = [...] }; public $server = class Symfony\Component\HttpFoundation\ServerBag { protected $parameters = [...] }; public $files = class Symfony\Component\HttpFoundation\FileBag { protected $parameters = [...] }; public $cookies = class Symfony\Component\HttpFoundation\ParameterBag { protected $parameters = [...] }; public $headers = class Symfony\Component\HttpFoundation\HeaderBag { protected $headers = [...]; protected $cacheControl = [...] }; protected $content = NULL; protected $languages = NULL; protected $charsets = NULL; protected $encodings = NULL; protected $acceptableContentTypes = NULL; protected $pathInfo = '/sell/orders/17301/view'; protected $requestUri =  
etc.. about 48 traces

**Comment by Phatdad76** [comment]
Firstly, thank you for responding and looking into this error. I check if debug mode is on and take a screenshot.  Thank you again 👍😊

**Comment by Phatdad76** [comment]
The error only appears on dashboard, I am able to use back office and make changes. It only appears upon login and clicking on dashboard.  Yes debug mode is on and saved.
![IMG_8331](https://github.com/user-attachments/assets/9b944846-eb28-44dc-9105-7a52f4dc291e)

**Comment by Phatdad76** [comment]
![IMG_8333](https://github.com/user-attachments/assets/68544241-2505-43b6-a78a-5a6acb3d335f)

**Comment by Phatdad76** [comment]
![IMG_8334](https://github.com/user-attachments/assets/5324fc1e-415b-4c81-80b3-389d11eb8767)

**Comment by bluegifts** [comment]
I can confirm i have same issue, same error in BO when debug mode on. Errors in php log on servet.

**Comment by Phatdad76** [comment]
Seems to run ok with debug mode off

**Comment by hschoenenberger** [comment]
Yes i'ts definetely related to debug mode, also be sure not to use development `error_reporting` configuration on a production environment. 
This will be fixed in the upcoming release, but please disable DEBUG mode  in the meanwhile.
Sorry again for the inconvenience.

**Comment by Phatdad76** [comment]
> Yes i'ts definetely related to debug mode, also be sure not to use development `error_reporting` configuration on a production environment. This will be fixed in the upcoming release, but please disable DEBUG mode in the meanwhile. Sorry again for the inconvenience.

Thank you, will do 👍

**Comment by hschoenenberger** [comment]
> With debug mode off error keep appearing on server php log

And that's expected, thoses are Warnings and Notices, you can tune error_repporting at server level

**Comment by jokesterfr** [comment]
The new 7.1 version is fixing this issue, please reopen if this does not resolve this exact issue.

Thank you,
